package.py

```diff
@@ -493,17 +480,14 @@ def __download_patched_pip(self) -> Path:
         :return:
         """
         pip_wheel = 'pip-22.1.2-py3-none-any.whl'
-        response = requests.get(f'https://github.com/rotki/rotki-build/raw/main/{pip_wheel}')
-        if response.status_code == 200:
-            temporary_directory = self.__storage.temporary_directory
-            temporary_directory.mkdir(exist_ok=True)
-            wheel_file = temporary_directory / pip_wheel
-            with open(wheel_file, 'wb') as file:
-                file.write(response.content)
-            return wheel_file
-
-        logger.error(f'pip failed to download {pip_wheel}')
-        raise Exception(f'{pip_wheel} download failed')
+        temporary_directory = self.__storage.temporary_directory
+        temporary_directory.mkdir(exist_ok=True)
+        wheel_file = temporary_directory / pip_wheel
+        filename, _ = urllib.request.urlretrieve(
+            f'https://github.com/rotki/rotki-build/raw/main/{pip_wheel}',
+            wheel_file,
+        )
```

Comment:
in the backend since it helps catching erros we try to be explicit with the arguments when we break the line.

```suggestion
        filename, _ = urllib.request.urlretrieve(
            url=f'https://github.com/rotki/rotki-build/raw/main/{pip_wheel}',
            filename=wheel_file,
        )
```